<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>loginWindow</class>
 <widget class="QMainWindow" name="loginWindow">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>320</width>
    <height>480</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>MainWindow</string>
  </property>
  <property name="styleSheet">
   <string notr="true">QPushButton#btn_dn:pressed{
	padding-left: 5px;
	padding-top: 5px;
	background-color:rgba(18,101,188,0.7);
}

QPushButton#btn_thoat_login:press{
	padding-left: 5px;
	padding-top: 5px;
	background-color:rgba(18,101,188,0.7);
}

QPushButton#btn_dn:hover{
	background-color:rgba(79,86,148,0.8);
}</string>
  </property>
  <widget class="QWidget" name="centralwidget">
   <widget class="QWidget" name="widget" native="true">
    <property name="geometry">
     <rect>
      <x>10</x>
      <y>10</y>
      <width>290</width>
      <height>410</height>
     </rect>
    </property>
    <widget class="QLabel" name="label">
     <property name="geometry">
      <rect>
       <x>0</x>
       <y>0</y>
       <width>290</width>
       <height>410</height>
      </rect>
     </property>
     <property name="styleSheet">
      <string notr="true">background-color:rgba(74, 78, 196, 1);
border-radius: 15px;
color: white;</string>
     </property>
     <property name="text">
      <string/>
     </property>
    </widget>
    <widget class="QLineEdit" name="txt_tk">
     <property name="geometry">
      <rect>
       <x>20</x>
       <y>210</y>
       <width>240</width>
       <height>30</height>
      </rect>
     </property>
     <property name="font">
      <font>
       <pointsize>10</pointsize>
      </font>
     </property>
     <property name="styleSheet">
      <string notr="true">background-color: rgba(0,0,0,0);
border-radius: 1px solid rgba(0,0,0,0);
border-bottom-color: rgba(255,255,255,1);
color: rgb(255,255,255);
padding-bottom: 7px ;</string>
     </property>
     <property name="placeholderText">
      <string> Tài khoản</string>
     </property>
    </widget>
    <widget class="QLineEdit" name="txt_mk">
     <property name="geometry">
      <rect>
       <x>20</x>
       <y>260</y>
       <width>240</width>
       <height>30</height>
      </rect>
     </property>
     <property name="font">
      <font>
       <pointsize>10</pointsize>
      </font>
     </property>
     <property name="styleSheet">
      <string notr="true">background-color: rgba(0,0,0,0);
border-radius: 1px solid rgba(0,0,0,0);
border-bottom-color: rgba(255,255,255,1);
color: rgb(255,255,255);
padding-bottom: 7px ;</string>
     </property>
     <property name="echoMode">
      <enum>QLineEdit::Password</enum>
     </property>
     <property name="placeholderText">
      <string> Mật khẩu</string>
     </property>
    </widget>
    <widget class="QPushButton" name="btn_dn">
     <property name="geometry">
      <rect>
       <x>20</x>
       <y>310</y>
       <width>240</width>
       <height>30</height>
      </rect>
     </property>
     <property name="font">
      <font>
       <pointsize>10</pointsize>
      </font>
     </property>
     <property name="cursor">
      <cursorShape>PointingHandCursor</cursorShape>
     </property>
     <property name="styleSheet">
      <string notr="true">background-color: rgba(79,86,146,1);
color: rgba(237,237,237,0.9);
border-radius: 10px;</string>
     </property>
     <property name="text">
      <string>Đ Ă N G N H Ậ P</string>
     </property>
    </widget>
    <widget class="QLabel" name="txt_login">
     <property name="geometry">
      <rect>
       <x>20</x>
       <y>80</y>
       <width>221</width>
       <height>91</height>
      </rect>
     </property>
     <property name="font">
      <font>
       <pointsize>40</pointsize>
      </font>
     </property>
     <property name="styleSheet">
      <string notr="true">color: rgba(255,255,255,0.7);</string>
     </property>
     <property name="text">
      <string>Login</string>
     </property>
    </widget>
    <widget class="QLabel" name="lb_tg">
     <property name="geometry">
      <rect>
       <x>20</x>
       <y>360</y>
       <width>55</width>
       <height>16</height>
      </rect>
     </property>
     <property name="font">
      <font>
       <weight>50</weight>
       <italic>false</italic>
       <bold>false</bold>
       <underline>true</underline>
       <strikeout>false</strikeout>
       <kerning>true</kerning>
      </font>
     </property>
     <property name="cursor">
      <cursorShape>PointingHandCursor</cursorShape>
     </property>
     <property name="styleSheet">
      <string notr="true">color: rgba(255,255,255,0.6);</string>
     </property>
     <property name="text">
      <string>Trợ giúp?</string>
     </property>
    </widget>
    <widget class="QPushButton" name="btn_thoat_login">
     <property name="geometry">
      <rect>
       <x>250</x>
       <y>0</y>
       <width>41</width>
       <height>28</height>
      </rect>
     </property>
     <property name="cursor">
      <cursorShape>PointingHandCursor</cursorShape>
     </property>
     <property name="styleSheet">
      <string notr="true">background-color:rgba(74, 78, 196, 0.8);
color: rgba(237,237,237,0.9);
border-radius: 15px;</string>
     </property>
     <property name="text">
      <string>X</string>
     </property>
    </widget>
   </widget>
  </widget>
  <widget class="QMenuBar" name="menubar">
   <property name="geometry">
    <rect>
     <x>0</x>
     <y>0</y>
     <width>320</width>
     <height>26</height>
    </rect>
   </property>
  </widget>
  <widget class="QStatusBar" name="statusbar"/>
 </widget>
 <resources/>
 <connections/>
</ui>
